### ISSUE TITLE
failure on meteor 1.3 release candidate

### ISSUE THREAD

**Issue by duffar** [frozeman/meteor-build-client#25]
It looks like meteor 1.3 clients require an additional property in the **meteor_runtime_config**.
`"meteorEnv":{"NODE_ENV":"production"}}`

If it doesn't find the property the client crashes at startup with a:

> Uncaught TypeError: Cannot read property 'NODE_ENV' of undefined

It seems unfortunate that the meteor client would rely on the NODE_ENV property and maybe its just an oversight on their part that will be resolved by the time 1.3 releases - but its broken at the moment none the less.

I guess passing the additional property is a pretty simple change however since 1.3 is not officially released it might be premature.

Perhaps an option to pass additional runtime config fields would allow us to work around issues like this without requiring further code changes. Once 1.3 is release a more permanent change to the property would be appropriate.

**Comment by jeffryan** [comment]
I should mention I tried supplying these as PUBLIC_SETTINGS via the settings.json but it didn't work. Perhaps an alternate section of the file could be used to supply additional properties. I realize there is risk of colliding with other things your doing with the runtime config but it would still be useful at times like this. If you'd like me to fork and make an upstream PR let me know...

**Comment by andrei-markeev** [comment]
This happens on official Meteor 1.3 also.
Since it is downloaded automatically, I assume many people will be having same issue.

The simple workaround - as @duffar correctly mentioned, is to add this line:
```
__meteor_runtime_config__.meteorEnv = { NODE_ENV: 'production' };
```
to the index.html, after `__meteor_runtime_config__` is assigned.

**Comment by frozeman** [comment]
a PR would be good :) 
https://github.com/frozeman/meteor-build-client/blob/master/meteor.js#L147-L155

**Comment by dr-dimitru** [comment]
Closed as fixed by #27
